docs/settings.md

```diff
@@ -56,41 +56,42 @@ Option legend:
+| `signaling_dev`                      | string<br>`yes` or `no`                                          | `no`                                                                      |        | Developer flag that allows to suppress various requirements like a Redis server when using the HPB                                                                                |
+| `breakout_rooms`                     | string<br>`yes` or `no`                                          | `yes`                                                                     |        | Whether or not breakout rooms are allowed (Will only prevent creating new breakout rooms. Existing conversations are not modified.)                                               |
+| `call_recording`                     | string<br>`yes` or `no`                                          | `yes`                                                                     |        | Enable call recording                                                                                                                                                             |
+| `federation_enabled`                 | string<br>`yes` or `no`                                          | `no`                                                                      |        | 🏗️ *Work in progress:* Whether or not federation with this instance is allowed                                                                                                   |
+| `conversations_files`                | string<br>`1` or `0`                                             | `1`                                                                       | 🖌️    | Whether the files app integration is enabled allowing to start conversations in the right sidebar                                                                                 |
+| `conversations_files_public_shares`  | string<br>`1` or `0`                                             | `1`                                                                       | 🖌️    | Whether the public share integration is enabled allowing to start conversations in the right sidebar on the public share page (Requires `conversations_files` also to be enabled) |
+| `enable_matterbridge`                | string<br>`1` or `0`                                             | `0`                                                                       | 🖌️    | Whether the matterbridge integration is enabled and can be configured                                                                                                             |
+| `allowed_recording_mimes`            | string[]                                                         | `{"audio\/ogg":["ogg"],"video\/ogg":["ogv"],"video\/x-matroska":["mkv"]}` |        | List of mime types and associated extensions of recording formats accepted by recording API                                                                                       |
```

Comment:
please move below `start_conversations` where `string[]` arrays are.
This list is sorted by "internal type"

Reply:
`allowed_recording_mimes` was removed.